package.json

```diff
@@ -20,24 +26,37 @@
       "url": "https://github.com/cowboy/node-findup-sync/blob/master/LICENSE-MIT"
     }
   ],
-  "main": "lib/findup-sync",
-  "engines": {
-    "node": ">= 0.6.0"
-  },
+  "files": [
+    "index.js"
+  ],
+  "main": "index.js",
   "scripts": {
-    "test": "grunt nodeunit"
+    "test": "grunt && mocha -R spec"
```

Comment:
`-R spec` isn't needed since the latest `mocha` uses `spec` reporter by default.